app/routes/_index.tsx

```diff
@@ -1,18 +1,50 @@
-import { type MetaFunction } from "@remix-run/node"
+import { json } from "@remix-run/node"
+
+import { useLoaderData } from "@remix-run/react"
+import { ContentEvents } from "~/components/contents/events"
 import { ContentIntro } from "~/components/contents/intro"
+import { ContentMembers } from "~/components/contents/members"
 import { BackgroundGradient } from "~/components/shared/background-gradient"
-import { createMeta } from "~/utils/meta"
+import { prisma } from "~/libs/db.server"
 import { createSitemap } from "~/utils/sitemap"
 
 export const handle = createSitemap("/", 1)
 
-export const meta: MetaFunction = () =>
-  createMeta({
-    title: "BandungDev",
-    description: "Developer community in Bandung, West Java, Indonesia.",
+export const loader = async () => {
+  // FIXME: Use Prisma transactions to concurrently get all 3 arrays
```

Comment:
@abui-am Check again for the codebase convention and how to use Prisma Transaction to retrieve more than 1 collection of records.

https://www.prisma.io/docs/orm/prisma-client/queries/transactions#the-transaction-api